pandas/tests/indexes/datetimes/test_tools.py

```diff
@@ -247,6 +247,55 @@ def test_to_datetime_parse_timezone_keeps_name(self):
 
 
 class TestToDatetime(object):
+    @pytest.mark.parametrize("s, _format, dt", [
```

Comment:
we should split up this file to segregate format parsing into a separate test file, cc @jbrockmendel, @mroeschke